src/datasource.js

```diff
@@ -74,31 +89,35 @@ export class HawkularDatasource {
   }
 
   testDatasource() {
+    // If tenants is unknown at this point (when having per-query tenants)
+    // We do a more basic check to status endpoint
+    // Else, it's full connectivity with tenant check
+    const endpoint = this.isTenantPerQuery ? '/status' : '/metrics';
```

Comment:
The thing is that the status endpoint might be reachable even without the proper credentials